packages/select/src/Select/Select.tsx

```diff
@@ -143,17 +152,27 @@ export const Select = forwardRef<HTMLDivElement, SelectProps>(
       }
     }, [onChange, readOnly, value]);
 
+    useEffect(() => {
+      if (openProp !== undefined && setOpenProp === undefined) {
+        console.warn(
```

Comment:
I think this should be an error, since the component won't behave as expected
```suggestion
        consoleOnce.error(
```